cla-backend-go/cla_manager/service.go

```diff
@@ -431,6 +432,17 @@ func sendRemovedClaManagerEmailToRecipient(companyModel *models.Company, project
 			whichEmail = companyAdmin.Emails[0]
 		}
 
+		// Try getting user email from userservice
+		userClient := v2UserService.GetClient()
+		if companyAdmin.LfUsername != "" {
+			lfUser, lfErr := userClient.GetUserByUsername(companyAdmin.LfUsername)
+			if lfErr != nil {
+				log.Warnf("unable to get user by username: %s , error: %+v ", companyAdmin.LfUsername, lfErr)
+			} else {
+				whichEmail = *lfUser.Emails[0].EmailAddress
```

Comment:
Harold, quick points:

- Is this bit of logic used anywhere else? If so, can we move this to a convenience function in the user_service.client.go file?
- Let's check to see if `lfUser` is nil and if `len(lfUser.Emails) > 0` before dereferencing - just to be safe

Reply:
Noted.. Shall update accordingly